connectors/src/connectors/zendesk/temporal/workflows.ts

```diff
@@ -378,12 +379,13 @@ async function runZendeskBrandHelpCenterSyncActivities({
     let cursor = null; // cursor involved in the pagination of the API
 
     while (hasMore) {
-      const result = await syncZendeskArticleBatchActivity({
+      const result = await syncZendeskArticlesBatchActivity({
```

Comment:
AFAIU we have to bump the workflow version to reflect this change

Reply:
`nit`: articles batch with an s sounds weird to me (book collection vs books collection).

Reply:
It's a "sync zendesk articles" batch activity no? 🤓

Reply:
oh yeah that works too, I read that as sync a batch of articles + activity as a suffix

Reply:
I prefer reading it as a "batch activity" actually, let's keep that!